src/gui/notificationwidget.cpp

```diff
@@ -44,6 +44,16 @@ void NotificationWidget::setActivity(const Activity &activity)
     _ui._subjectLabel->setVisible(!activity._subject.isEmpty());
     _ui._messageLabel->setVisible(!activity._message.isEmpty());
 
+    QString subject = activity._subject;
+    if( !activity._link.isEmpty() ) {
+        // append a link to the message, if that is empty, to subject
+        subject.append(QString("&nbsp;<a href=\"%1\">%2</a>")
+                       .arg(activity._link.toString(QUrl::FullyEncoded))
```

Comment:
use arg(... , ...)  with two arguments.   In general this is better because it works even if the first string contains a %1 or a %2.

Also, please use escapeXml  to the _linkText